src/styles/index.scss

```diff
@@ -1,3 +1,132 @@
+$hoverBlueColor: #00acdc;
+$hoverWhiteColor: #fff;
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
```

Comment:
Checklist:
2. [STYLES] - Remember to use fallback fonts - alternative font-family in case the main one doesn't work [like this](https://www.w3schools.com/cssref/pr_font_font-family.asp)
![image](https://github.com/mate-academy/layout_product-cards/assets/63243136/57df8758-073a-45f8-b56e-ba7f2cfe814d)